src/pages/Delegate/index.tsx

```diff
@@ -109,14 +111,35 @@ export const Delegate = () => {
       if (!tx) return
       ;(await tx)
         .signSubmitAndWatch(selectedAccount?.polkadotSigner)
-        .forEach((value) => console.log('value', value))
+        .forEach((value) => {
+          let msg: string
+          switch (value.type) {
+            case 'finalized':
+              msg = `Tx - Finalized. Block Number: ${value.block.number}. TxHash: ${value.txHash}`
+              break
+            case 'signed':
+              msg = `Tx - Signed. TxHash: ${value.txHash}`
```

Comment:
I missed that part sorry, this is definitely what I thought about originally, minus the tx hash that's really not great to read.I'd go really simple. We can get fancy with some icons etc in the future if we want.
```suggestion
              msg = "Tx signed."
```
Same for the others. These messages should ideally be kept small. Eventually with a link (on an icon) to an explorer, but a hash isn't really user friendly.